libs/sdk-backend-base/src/customBackend/workspace.ts

```diff
@@ -41,6 +41,10 @@ export class CustomWorkspace implements IAnalyticalWorkspace {
         throw new NotSupported("getting parent workspace is not supported");
     }
 
+    public setLocale(_locale: string): Promise<void> {
+        throw new NotSupported("Workspace Locale is not supported");
```

Comment:
When is this backend impl used?

Reply:
It is something that is required for build. Otherwise it fails as it is defined within `IAnalyticalWorkspace`. To my understanding, it should be required for SPI.